controller/src/main/java/io/pravega/controller/store/stream/VersionedTransactionData.java

```diff
@@ -20,6 +20,9 @@
 @Data
 @AllArgsConstructor
 public class VersionedTransactionData {
+    public static final VersionedTransactionData NULL = new VersionedTransactionData(Integer.MIN_VALUE, new UUID(0, 0),
```

Comment:
`NULL` is really overloaded, can we use something different? a few suggested options: `EMPTY`, `INVALID`, `VAPID`.